packages/react-reconciler/src/ReactFiberCommitWork.new.js

```diff
@@ -2061,7 +2094,11 @@ function commitMutationEffects_complete(root: FiberRoot) {
   }
 }
 
-function commitMutationEffectsOnFiber(finishedWork: Fiber, root: FiberRoot) {
+function commitMutationEffectsOnFiber(
+  finishedWork: Fiber,
+  root: FiberRoot,
+  committedLanes: Lanes,
```

Comment:
It would be better to store this on a module variable so we don't have to pay the cost of the extra writing to the stack in prod.

Reply:
Sure.

Seems a little arbitrary, since we're already passing around at least the root to most of these functions, but I'll make the minimal change for now and use module scope for the root and lanes.

Reply:
Done.